src/main/java/nl/dtls/fairdatapoint/entity/index/settings/IndexSettings.java

```diff
@@ -46,10 +46,14 @@ public class IndexSettings {
     @NotNull
     private IndexSettingsPing ping;
 
+    @NotNull
+    private Boolean autoPermit;
+
     public static IndexSettings getDefault() {
         final IndexSettings settings = new IndexSettings();
         settings.setPing(IndexSettingsPing.getDefault());
         settings.setRetrieval(IndexSettingsRetrieval.getDefault());
+        settings.setAutoPermit(true);
```

Comment:
Set to `true` by default; is this something that can be defined in a config file too?

Reply:
OK, will do... but default from the file will be also true right? For the compatibility & expected behaviour...